frontend/src/components/App.js

```diff
@@ -16,63 +19,66 @@ class App extends React.Component {
         }
     }
 
+    abortController = new window.AbortController();
+
     componentDidMount() {
-        fetch("http://localhost:5000/api/calendar/", {
-            method: 'POST',
-            body: JSON.stringify(this.state.start)
-        }) //todo move to prop file
+        let url = new URL(ADDRESS + API);
```

Comment:
lets extract fetching events in method so we don't repeat code

Reply:
Fixed.